crates/egui/src/context.rs

```diff
@@ -489,7 +489,11 @@ impl ContextImpl {
                 &viewport.prev_frame.widgets,
                 &viewport.hits,
                 &viewport.input,
-                self.memory.interaction_mut(),
+                self.memory
+                    .interactions
+                    .entry(self.memory.viewport_id)
+                    .or_default(),
+                &self.memory.options,
```

Comment:
This isn't super pretty, but otherwise it conflicts with the mut access of `memory` above